spec-0000/index.md

```diff
@@ -32,8 +32,8 @@ All versions refer to feature releases (i.e., Python 3.8.0, NumPy 1.19.0; not Py
 
 Specifically, we recommend that:
 
-1. Support for Python versions be dropped **3 years** after their initial release.
-2. Support for core package dependencies be dropped **2 years** after their initial release.
+1. Support for Python versions be dropped **3 years** after the next version of Python is released.
+2. Support for core package dependencies be dropped **2 years** after the next version of the package is released.
 
 {{< admonition note >}}
```

Comment:
If I understand the admonition correctly, the recommendation is to provide at least the indicated support window and not much more, but we recognize that there may be extreme circumstances in which support for a core dependency must be dropped early (e.g. to support a critical bug fix). Since it is always true that projects may endorse a SPEC as good practice but not follow it for any reason, I'm not sure that the admonition is needed unless it's particularly important to highlight this example.